tox.ini

```diff
@@ -31,6 +31,10 @@ commands = pylint -f colorized -E google test --ignore=src-gen
 deps = pylint
 commands = pylint -f colorized -d all -e W -r n google test --ignore=src-gen
 
+[testenv:pylint-no-test-deps]
+deps = pylint
```

Comment:
From the output, it looks like it builds and installs the `gax-python` package itself before running pylint (the effect being that it pulls in the dependencies via the list in `setup.py`.)

Given that, it might actually be unnecessary to install `requirements.txt` in the other pylint environments. I will look into it.